### ISSUE TITLE
VS Code formatting issue

### ISSUE THREAD

**Issue by pramodya1994** [ballerina-platform/ballerina-lang#19242]
**Description:**
When formatting below code in VS Code:
```ballerina
map<json> recMap = check map<json>.constructFrom( rec);
```
After formatting:
```ballerina
map<json> recMap = checkmap<json>.constructFrom( rec);
```

**Steps to reproduce:**

**Affected Versions:**

**OS, DB, other environment details and versions:**

**Related Issues (optional):**

**Suggested Labels (optional):**

**Suggested Assignees (optional):**

**Comment by NipunaMarcus** [comment]
This is fixed in the 1.0.1 version of the Ballerina ... @pramodya1994 are you in a position to upgrade to Ballerina 1.0.1 version ... ?

**Comment by pramodya1994** [comment]
Great, Then we can close it.

**Comment by NipunaMarcus** [comment]
This has been fixed with Ballerina v1.0.1 hence closing the issue.
